docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -6,64 +6,89 @@ This feature is supported in mobile and wearable applications only.
 
 The main features of the Bluetooth API include:
 
```

Comment:
I would add a section about handling UUIDs, binary data in all APIs and new functions that can be used to convert these things between different formats, we added in 6.0. I think, the new section can go before anything else in this guide, because it applies to BLE and Bluetooth Classic.